modules/autofill/autofill-store/build.gradle

```diff
@@ -41,27 +13,45 @@ dependencies {
     implementation project(path: ':autofill-api')
     implementation project(path: ':secure-storage-api')
 
-    implementation 'androidx.core:core-ktx:1.7.0'
-    implementation 'androidx.appcompat:appcompat:1.4.1'
+    implementation AndroidX.core.ktx
+    implementation AndroidX.appCompat
     implementation KotlinX.coroutines.core
     implementation Google.android.material
     implementation project(path: ':secure-storage-api')
+
+    implementation Google.dagger
+    implementation JakeWharton.timber
+
+    // Testing dependencies
+    testImplementation AndroidX.work.testing
```

Comment:
Why are we adding all this new modules when we haven't added any thing new in the tests in this module?

Reply:
I was having problems with the unit tests failing on this module, and re-structuring the build.gradle was part of my attempt at fixing it. I believe it was likely a local cacheing issue, so reverted some of these changes. Kept a few others that seemed like a good improvement anyways.